G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelAddSheet/ShouldAddSheet_WhenExcelAddSheetCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,14 @@
+﻿excel.open
+
+for counter ♥a from 1 to 3 step 1
+excel.addsheet name ‴TestSheet♥a‴
```

Comment:
Missing indentation here and below. `Step` parameter with value `1` is redundant, same with `counter`.
So `for ♥a from 1 to 3`. Please change also other places where you use "for" loop.